normandy/recipes/migrations/0027_auto_20160506_1944.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
depending on whose PR lands first (this one or #141) this migration might cause a conflict.